.github/workflows/pull-request.yml

```diff
@@ -8,10 +8,30 @@ on:
     branches:
       - main
 
+env:
+  RETENTION_DAYS: 7
+
+permissions:
+  contents: read
```

Comment:
Why do you need to give this permissions?

Reply:
This is required because of get-playwright-version job
Without this Github shows security warning (please check in the history of Conversation)